lms/djangoapps/discussion_api/api.py

```diff
@@ -274,6 +275,7 @@ def get_thread_list(
         "last_activity_at".
     order_direction: The direction in which to sort the threads by. The only
         values are "asc" or "desc". The default is "desc".
+    discussion_id: ID of an inline discussion for any module
```

Comment:
This is true actually because inline discussions are also against course; its only one additional optional filter parameter for this generic endpoint.